helm_deploy/prison-staff-hub/templates/_envs.tpl

```diff
@@ -85,6 +85,9 @@ env:
   - name: NN_ENDPOINT_URL
     value: {{ .Values.env.NN_ENDPOINT_URL | quote }}
 
+  - name: CASENOTES_API_URL
+    value: {{ .Values.env.CASENOTES_API_URL | quote }}
```

Comment:
values-dev, values-preprod and values-prod have already been updated in a previous PR.